src/core/server.js

```diff
@@ -800,7 +810,10 @@ Server.prototype._sendMessage = function(message) {
     if (this._remote.trace) {
       log.info(this.getServerID(), 'request:', message);
     }
-    this._ws.send(JSON.stringify(message));
+    this._ws.send(JSON.stringify(message), () => {
+      // sometimes gitves 'not opened'
```

Comment:
Shouldn't this generate an error? It looks like it's just silently ignoring the problem.